### PR TITLE
Updates to disable-tls-1.0-1.1.md

### PR DESCRIPTION
**Pull request by robertpp**
Changes:
- Updated the requirement for Skype for Business Server 2019 to CU1.
- Updated the requirement for Skype for Business Server 2015 to CU9. 

**Note**: These two changes might need to be reverted and added as a mentions, since all the subsequent procedures were written for Skype for Business Server 2015 CU6 HF2 and Skype for Business Server 2019.

- Moved Lync Room System (a.k.a. SRSv1) to 'Out of scope'.
- Moved Microsoft Teams Rooms and Surface Hub from to 'Fully tested'.
- Removed SBA and SBS since already documented as 'Fully tested'.
- Miscellaneous formatting changes.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 1]
```diff
@@ -30,55 +30,46 @@ Microsoft has produced a white paper on TLS available [here](https://cloudblogs.
...
 
 ### Fully tested and supported servers
 
-- Skype for Business Server 2019
-- Skype for Business Server 2015 CU6 HF2 6.0.9319.516 ([March 2018 update](https://support.microsoft.com/en-us/help/4086059/march-2018-cumulative-update-6-0-9319-516-for-skype-for-business)) and higher on: 
-    - Windows Server 2012 (with KB 3140245 or superseding update), 2012 R2 or 2016
-- In-place Upgraded Skype for Business Server 2015, with CU6 HF2 and higher on 
-    - Windows Server 2008 R2, 2012 (with KB [3140245](https://support.microsoft.com/en-us/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in) or superseding update), or 2012 R2
+- Skype for Business Server 2019 CU1 17.0.2046.123 (June 2019) or higher
+- Skype for Business Server 2015 CU9 6.0.9319.548 (May 2019) or higher on Windows Server 2012 (with KB KB [3140245](https://support.microsoft.com/en-us/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in) or superseding update), 2012 R2 or 2016.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
- Skype for Business Server 2015 CU9 6.0.9319.548 (May 2019) or higher on Windows Server 2012 (with KB [3140245](https://support.microsoft.com/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in) or superseding update), 2012 R2 or 2016.
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 2]
```diff
@@ -30,55 +30,46 @@ Microsoft has produced a white paper on TLS available [here](https://cloudblogs.
...
+- In-place Upgraded Skype for Business Server 2015, with CU9 6.0.9319.548 (May 2019) or higher on Windows Server 2008 R2, 2012 (with KB [3140245](https://support.microsoft.com/en-us/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in) or superseding update), or 2012 R2.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
- In-place Upgraded Skype for Business Server 2015, with CU9 6.0.9319.548 (May 2019) or higher on Windows Server 2008 R2, 2012 (with KB [3140245](https://support.microsoft.com/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in) or superseding update), or 2012 R2.
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 3]
```diff
@@ -30,55 +30,46 @@ Microsoft has produced a white paper on TLS available [here](https://cloudblogs.
...
 - Survivable Branch Appliance (SBA) with Skype for Business Server 2015 CU6 HF2 or higher (confirm with your vendor that they packaged the appropiate updates and have been made available for your appliance)
 - Survivable Branch Server (SBS) with Skype for Business Server 2015 CU6 HF2 or higher
 - Lync Server 2013 **Edge Role Only**, this is because Edge role does not have a dependency on Windows Fabric 1.0.
 
-
 ### Fully tested and supported clients
 
-- Lync 2013 (Skype for Business) Desktop Client, MSI and C2R, including Basic [15.0.5023.1000 and higher](https://support.microsoft.com/en-us/help/4018334/april-3-2018-update-for-skype-for-business-2015-lync-2013-kb4018334)
-- Skype for Business 2016 Desktop Client, MSI [16.0.4678.1000 and higher](https://support.microsoft.com/en-us/help/4018323/april-3-2018-update-for-skype-for-business-2016-kb4018323), including Basic
+- Lync 2013 (Skype for Business) Desktop Client, MSI and C2R, including Basic [15.0.5023.1000 or higher](https://support.microsoft.com/en-us/help/4018334/april-3-2018-update-for-skype-for-business-2015-lync-2013-kb4018334)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
- Lync 2013 (Skype for Business) Desktop Client, MSI and C2R, including Basic [15.0.5023.1000 or higher](https://support.microsoft.com/help/4018334/april-3-2018-update-for-skype-for-business-2015-lync-2013-kb4018334)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 4]
```diff
@@ -30,55 +30,46 @@ Microsoft has produced a white paper on TLS available [here](https://cloudblogs.
...
+- Skype for Business 2016 Desktop Client, MSI [16.0.4678.1000 or higher](https://support.microsoft.com/en-us/help/4018323/april-3-2018-update-for-skype-for-business-2016-kb4018323), including Basic
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
- Skype for Business 2016 Desktop Client, MSI [16.0.4678.1000 or higher](https://support.microsoft.com/help/4018323/april-3-2018-update-for-skype-for-business-2016-kb4018323), including Basic
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 5]
```diff
@@ -119,15 +110,15 @@ PIC (Public IM Connectivity) to Skype Consumer service: We do not expect disabli
...
+At a high level, while Skype for Business Server 2019 is ready for procedure at install, Skype for Business Server 2015 will require that you install CU9, applying pre-requisite updates to .NET and SQL, deploying prerequisite registry keys, and finally a separate round of OS configuration updates (i.e. disabling TLS 1.0 and 1.1 via registry file import). It is critically important that you complete installation of all prerequisites, including Skype for Business Server 2015 CU6 HF2, prior to disabling TLS 1.0 and 1.1 on any server in your environment. Every Skype for Business server, including Edge role and SQL Backends, requires the updates. Also ensure that all supported (in-scope) clients have been updated to the required minimum versions. Don’t forget to update management workstations as well.
 
 We want to follow the usual order of operations of "inside out" for upgrading Skype for Business servers. Treat Director pools, Persistent chat, and Paired Pools in the same manner you normally would. Order and methods for upgrade are covered [here](topology.md) and [here](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
We want to follow the usual order of operations of "inside out" for upgrading Skype for Business servers. Treat Director pools, Persistent chat, and Paired Pools in the same manner you normally would. Order and methods for upgrade are covered [here](topology.md) and [here](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015).
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 6]
```diff
@@ -141,7 +132,7 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
 
 Extensive dependency updating is required before you begin to disable TLS 1.0 and 1.1 at the operating system level in your Skype for Business Server 2015 deployments. The following are the minimum versions that can support TLS 1.2. Deploy all prerequisite updates across every Skype for Business server in your environment before you begin disabling TLS 1.0 and 1.1.
 
-- Skype for Business Server 2015 CU6 HF2 6.0.9319.516 ([March 2018 update](https://support.microsoft.com/en-us/help/4086059/march-2018-cumulative-update-6-0-9319-516-for-skype-for-business)) or higher
+- Skype for Business Server 2015 CU9 6.0.9319.548 (May 2019) or higher
 - [.NET Framework 4.7](https://www.microsoft.com/en-us/download/details.aspx?id=55167) or higher with SchUseStrongCrypto enabled in the registry (provided below)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
- [.NET Framework 4.7](https://www.microsoft.com/download/details.aspx?id=55167) or higher with SchUseStrongCrypto enabled in the registry (provided below)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 7]
```diff
@@ -141,7 +132,7 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
 - SQL must be updated on all Skype for Business 2015 servers and backends. Update Enterprise Edition Pool SQL Backends first, then their respective FEs. 
     - SQL Server 2014 SP1 + CU5 ([link](https://support.microsoft.com/help/3130926)), or higher / SQL Server 2012 SP2 + CU16 or higher/ SQL Server 2014 RTM + CU12 ([link](https://support.microsoft.com/en-us/help/3130923/cumulative-update-12-for-sql-server-2014)) or higher / SQL Server 2014 SP2
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - SQL Server 2014 SP1 + CU5 ([link](https://support.microsoft.com/help/3130926)), or higher / SQL Server 2012 SP2 + CU16 or higher/ SQL Server 2014 RTM + CU12 ([link](https://support.microsoft.com/help/3130923/cumulative-update-12-for-sql-server-2014)) or higher / SQL Server 2014 SP2
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 8]
```diff
@@ -152,16 +143,16 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
 
 ### Basic steps to install pre-requisites, in recommended order of operations
 
-1. Install the Skype for Business Server CU6HF2 (6.0.9319.516) update to all servers. 
+1. Install the Skype for Business Server CU9 update to all servers. 
     1. Install the update to components using the updater.
-    2. Update databases according to documented procedures. Instructions are documented at [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015).
+    2. Update databases according to documented procedures. For Skype for Business Server 2015, see KB [3061064](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    2. Update databases according to documented procedures. For Skype for Business Server 2015, see KB [3061064](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015).
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 9]
```diff
@@ -152,16 +143,16 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     3. Validate product functionality in the deployment prior to moving forward with any other changes.
 2. Download .NET 4.7 Offline Installer. 
     1. Reference: [https://www.microsoft.com/en-us/download/details.aspx?id=55167](https://www.microsoft.com/en-us/download/details.aspx?id=55167)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://www.microsoft.com/download/details.aspx?id=55167](https://www.microsoft.com/en-us/download/details.aspx?id=55167)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 10]
```diff
@@ -152,16 +143,16 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     2. Ensure that Skype for Business Server 2015 services are stopped on the Front End server.
     3. Reference: [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    3. Reference: [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 11]
```diff
@@ -170,27 +161,27 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
         - Reference: [https://www.microsoft.com/en-us/download/details.aspx?id=53168](https://www.microsoft.com/en-us/download/details.aspx?id=53168)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
        - Reference: [https://www.microsoft.com/download/details.aspx?id=53168](https://www.microsoft.com/en-us/download/details.aspx?id=53168)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 12]
```diff
@@ -170,27 +161,27 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     3. Copy the installation media to a folder on the server (Ex: C:\01_2014SqlSp2)
     4. Ensure Skype for Business Server 2015 services are stopped on the Front End server 
-        - Ex (Standard Edition): Stop-CsWindowsService
-        - Ex (Enterprise Edition): Invoke-CsComputerFailove
+        - Ex (Standard Edition): ```Stop-CsWindowsService```
+        - Ex (Enterprise Edition): ```Invoke-CsComputerFailover```
     5. Open an Admin Command Prompt, and upgrade all installed components and instances 
         - Example: C:\01_2014SqlSp2\SQLServer2014SP2-KB3171021-x64-ENU.exe /qs /IAcceptSQLServerLicenseTerms /Action=Patch /AllInstances
 4. Update SQL Native Client. 
     1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 13]
```diff
@@ -170,27 +161,27 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     2. Download from [https://www.microsoft.com/en-us/download/details.aspx?id=50402](https://www.microsoft.com/en-us/download/details.aspx?id=50402)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    2. Download from [https://www.microsoft.com/download/details.aspx?id=50402](https://www.microsoft.com/en-us/download/details.aspx?id=50402)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 14]
```diff
@@ -170,27 +161,27 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     4. Stop the SQL instances installed from running 
-        - Ex: Get-Service 'MSSQL$RTCLOCAL' | Stop-Servic
-        - Ex: Get-Service 'MSSQL$LYNCLOCAL' | Stop-Servic
-        - Ex (Standard Edition Only): Get-Service 'MSSQL$RTC' | Stop-Servic
+        - Ex: ```Get-Service 'MSSQL$RTCLOCAL' | Stop-Service```
+        - Ex: ```Get-Service 'MSSQL$LYNCLOCAL' | Stop-Service```
+        - Ex (Standard Edition Only): ```Get-Service 'MSSQL$RTC' | Stop-Service```
     5. Install the update.
 5. Update ODBC Driver 11 for SQL Server. 
     1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 15]
```diff
@@ -170,27 +161,27 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
     2. Download from [https://www.microsoft.com/en-us/download/confirmation.aspx?id=36434](https://www.microsoft.com/en-us/download/confirmation.aspx?id=36434)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    2. Download from [https://www.microsoft.com/download/confirmation.aspx?id=36434](https://www.microsoft.com/download/confirmation.aspx?id=36434)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 16]
```diff
@@ -490,7 +481,7 @@ Because some dependency prerequisites are required to support TLS 1.2 in Skype f
 
 1. [Install prerequisites for Skype for Business Server](../../deploy/install/install-prerequisites.md)  
 2. 2. Install .NET 4.7: 
-      - **Note:** We first introduced support for .NET 4.7 in Skype for Business Server 2015 CU5+ (6.0.9319.281). Therefore, in later steps below we will be updating Core Components prior to the main install.
+      - **Note:** We first introduced support for .NET 4.7 in Skype for Business Server 2015 CU5 (6.0.9319.281). Therefore, in later steps below we will be updating Core Components prior to the main install.
       - Download: https://www.microsoft.com/en-us/download/details.aspx?id=55167.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
      - Download: https://www.microsoft.com/download/details.aspx?id=55167.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
@robertpp - can you commit @JohanFreelancer9's copy edit changes?  After that, I'll merge the changes. Thank you.

**Comment by robertpp** [comment]
Hi @LanaChin. All locales removal from links suggested by @JohanFreelancer9 have been committed.

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 17]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
+- Skype for Business Server 2015 CU9 6.0.9319.548 (May 2019) or higher
+- [.NET Framework 4.7](https://www.microsoft.com/download/details.aspx?id=55167) or higher with SchUseStrongCrypto enabled in the registry (provided below)
 - SQL must be updated on all Skype for Business 2015 servers and backends. Update Enterprise Edition Pool SQL Backends first, then their respective FEs. 
-    - SQL Server 2014 SP1 + CU5 ([link](https://support.microsoft.com/help/3130926)), or higher / SQL Server 2012 SP2 + CU16 or higher/ SQL Server 2014 RTM + CU12 ([link](https://support.microsoft.com/en-us/help/3130923/cumulative-update-12-for-sql-server-2014)) or higher / SQL Server 2014 SP2
+    - SQL Server 2014 SP1 + CU5 ([link](https://support.microsoft.com/help/3130926)), or higher / SQL Server 2012 SP2 + CU16 or higher/ SQL Server 2014 RTM + CU12 ([link](https://support.microsoft.com/help/3130923/cumulative-update-12-for-sql-server-2014)) or higher / SQL Server 2014 SP2
     - SQL Server Native Client for SQL Server 2012 ([link](https://www.microsoft.com/en-us/download/details.aspx?id=50402))
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - SQL Server Native Client for SQL Server 2012 ([link](https://www.microsoft.com/download/details.aspx?id=50402))
```

Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - SQL Server Native Client for SQL Server 2012 ([link](https://www.microsoft.com/download/details.aspx?id=50402))
```

Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - SQL Server Native Client for SQL Server 2012 ([link](https://www.microsoft.com/download/details.aspx?id=50402))
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 18]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
     - Microsoft ODBC Driver 11 for SQL Server ([link](https://www.microsoft.com/en-us/download/details.aspx?id=36434)), or higher
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - Microsoft ODBC Driver 11 for SQL Server ([link](https://www.microsoft.com/download/details.aspx?id=36434)), or higher
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 19]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
     - Shared Management Objects for SQL Server 2014 SP2 ([link](https://www.microsoft.com/en-in/download/details.aspx?id=42295))
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - Shared Management Objects for SQL Server 2014 SP2 ([link](https://www.microsoft.com/download/details.aspx?id=42295))
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 20]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
     - SQLSysClrTypes for SQL server 2014 SP2 ([link](https://www.microsoft.com/en-in/download/details.aspx?id=42295))
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - SQLSysClrTypes for SQL server 2014 SP2 ([link](https://www.microsoft.com/download/details.aspx?id=42295))
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 21]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
 
-1. Install the Skype for Business Server CU6HF2 (6.0.9319.516) update to all servers. 
+1. Install the Skype for Business Server CU9 update to all servers. 
     1. Install the update to components using the updater.
-    2. Update databases according to documented procedures. Instructions are documented at [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015).
+    2. Update databases according to documented procedures. For Skype for Business Server 2015, see KB [3061064](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015).
     3. Validate product functionality in the deployment prior to moving forward with any other changes.
 2. Download .NET 4.7 Offline Installer. 
-    1. Reference: [https://www.microsoft.com/en-us/download/details.aspx?id=55167](https://www.microsoft.com/en-us/download/details.aspx?id=55167)
+    1. Reference: [https://www.microsoft.com/download/details.aspx?id=55167](https://www.microsoft.com/en-us/download/details.aspx?id=55167)
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://www.microsoft.com/download/details.aspx?id=55167](https://www.microsoft.com/download/details.aspx?id=55167)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 22]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
     2. Ensure that Skype for Business Server 2015 services are stopped on the Front End server.
-    3. Reference: [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015)
-    4. Ex (Standard Edition): Stop-CsWindowsServices
-    5. Ex (Enterprise Edition): Invoke-CsComputerFailover
+    3. Reference: [https://support.microsoft.com/en-us/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015)
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    3. Reference: [https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015](https://support.microsoft.com/help/3061064/updates-for-skype-for-business-server-2015)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 23]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
+    4. Ex (Standard Edition): ```Stop-CsWindowsService```
+    5. Ex (Enterprise Edition): ```Invoke-CsComputerFailover```
     6. Run the installer package.
     7. Reboot the server.
 3. Update SQL Express 2014 on all servers. 
     1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server)
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 24]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
     2. Download SQL 2014 SP2 
-        - Reference: [https://www.microsoft.com/en-us/download/details.aspx?id=53168](https://www.microsoft.com/en-us/download/details.aspx?id=53168)
+        - Reference: [https://www.microsoft.com/download/details.aspx?id=53168](https://www.microsoft.com/en-us/download/details.aspx?id=53168)
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
        - Reference: [https://www.microsoft.com/download/details.aspx?id=53168](https://www.microsoft.com/download/details.aspx?id=53168)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 25]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
-        - Ex (Standard Edition): Stop-CsWindowsService
-        - Ex (Enterprise Edition): Invoke-CsComputerFailove
+        - Ex (Standard Edition): ```Stop-CsWindowsService```
+        - Ex (Enterprise Edition): ```Invoke-CsComputerFailover```
     5. Open an Admin Command Prompt, and upgrade all installed components and instances 
         - Example: C:\01_2014SqlSp2\SQLServer2014SP2-KB3171021-x64-ENU.exe /qs /IAcceptSQLServerLicenseTerms /Action=Patch /AllInstances
 4. Update SQL Native Client. 
-    1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server).
-    2. Download from [https://www.microsoft.com/en-us/download/details.aspx?id=50402](https://www.microsoft.com/en-us/download/details.aspx?id=50402)
+    1. Reference: [https://support.microsoft.com/en-us/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server).
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 26]
```diff
@@ -141,56 +132,56 @@ We want to follow the usual order of operations of "inside out" for upgrading Sk
...
+    2. Download from [https://www.microsoft.com/download/details.aspx?id=50402](https://www.microsoft.com/en-us/download/details.aspx?id=50402)
```
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    2. Download from [https://www.microsoft.com/download/details.aspx?id=50402](https://www.microsoft.com/download/details.aspx?id=50402)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md (hunk 23 above)
Comment (nenonix) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    1. Reference: [https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server](https://support.microsoft.com/help/3135244/tls-1-2-support-for-microsoft-sql-server)
```

**Comment by robertpp** [comment]
Committed the locale changes proposed by @nenonix.

**Pull request merged**